src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  let totalCost = days * costPerDay;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, the function uses hardcoded values for the cost per day, and the discounts for different rental durations. It's better to define these values as constants at the beginning of the function to make the code more maintainable and self-explanatory.